javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,26 @@
+//Housey-Pricey
+let volumeInMeters;
+let gardenSizeInM2;
+let additionalCost;
+let originalCost;
+let housePrice = (volumeInMeters * 2.5 * 1000) + (gardenSizeInM2 * 300);
+//Peter housePrice
+{
+    let volumeInMeters = 8 * 10 * 10;
+    let gardenSizeInM2 = 100;
+    let originalCost = 2500000;
+    let housePrice = (volumeInMeters * 2.5 * 1000) + (gardenSizeInM2 * 300);
+    let additionalCost = originalCost - housePrice;
+    console.log(originalCost < housePrice);
+    console.log('Peter will be paying an additional cost of ' + additionalCost + ' on the house');
+}
```

Comment:
I wouldn't use the curly brackets like this. 
You can remove them and get the same results.

Reply:
Since you're checking Peter's cost, you should use an if/else-statement:
```javascript
if (originalCost < housePrice) {
    console.log('Peter will be paying an additional cost of ' + additionalCost + ' on the house');
} else {
    console.log('Peter is paying a discount of ' + additionalCost + ' on the house');
}
```